client/shipping-label/components/label-refund-modal/index.js

```diff
@@ -20,36 +21,34 @@ const RefundDialog = ( { refundDialog, labelActions, storeOptions, created, refu
 		<Modal
 			isVisible={ refundDialog && refundDialog.labelId === label_id }
 			onClose={ labelActions.closeRefundDialog }
-			additionalClassNames="wcc-shipping-label-refund">
-			<div className="wcc-shipping-label-refund__content">
```

Comment:
The child rules it had, I put them directly under the `wcc-shipping-label-refund` rule. Same for the Reprint dialog.